app-shells/tmux-bash-completion/tmux-bash-completion-0_pre20200419.ebuild

```
@@ -0,0 +1,26 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit bash-completion-r1
+COMMIT="f5d53239f7658f8e8fbaf02535cc369009c436d6"
+DESCRIPTION="bash-completion scripts for tmux"
+HOMEPAGE="https://github.com/imomaliev/tmux-bash-completion"
+SRC_URI="https://github.com/imomaliev/tmux-bash-completion/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+RDEPEND="
+	>=app-misc/tmux-2.2
+	app-shells/bash-completion
+"
+DEPEND=""
+S="${WORKDIR}/${PN}-${COMMIT}"
+
+src_install() {
+	default
+	dobashcomp "${S}/completions/tmux"
+}
```

You might ant to add a `pkg-postinst` with a helpful message about sourcing the correct file. something like:

```suggestion
}

pkg-postinst{
    elog "Add the following line to ~/.bashrc to enable completions"
    elog
    elog "    # source ${EPREFIX}/usr/share/bash-completion/completions/tmux"
}
```